libs/common/src/types/mongo-db-models/tfm-activity.ts

```diff
@@ -0,0 +1,27 @@
+import { ACTIVITY_TYPES } from '../../constants';
+import { UnixTimestamp, UnixTimestampSeconds } from '../date';
+
+export type TfmActivity =
```

Comment:
This is a part of the TfmDeals model (stored in the database) so it could be used everywhere - in my next PR I'll use this type to update the database with a cancellation